ui/imports/utils/Constants.qml

```diff
@@ -1058,10 +1058,12 @@ QtObject {
     }
 
     enum TokenType {
```

Comment:
How about adding a note that this should be in sync with `app_service/common/types.nim`?

Reply:
sure let me do that.